EIPS/eip-4762.md

```diff
@@ -115,7 +115,7 @@ We define **write events** as follows. Note that when a write takes place, an ac
 
 #### Write events for account headers
 
-When a nonzero-balance-sending `*CALL` or `SELFDESTRUCT` with a given sender and recipient takes place, process these write events:
+When a nonzero-balance-sending `CALL` or `SELFDESTRUCT` with a given sender and recipient takes place, process these write events:
```

Comment:
In this EIP `*CALL` refers to `CALL`, `CALLCODE`, `DELEGATECALL` and `STATICCALL`, but:
- Value-bearing `CALLCODE` doesn't produce any effective transfer since the context contract is still the same one that executes the instruction.
- `DELEGATECALL` and `STATICCALL` can't be value bearing.

This means that this rule only makes sense for `CALL`.